### PR TITLE
Check and fix typos in CI

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
We now verify any typos in CI and fix them.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Fixed API typos in golang client.
```

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -1017,9 +1017,9 @@ func (c *ConmonClient) metadataBytes(ctx context.Context) ([]byte, error) {
 	return metadata, nil
 }
 
-// CreateaNamespacesConfig is the configuration for calling the
+// CreateNamespacesConfig is the configuration for calling the
```
Comment (haircommander) [inline pkg/client/client.go:1020]: do we need type aliases to keep backward compatibility?

Comment (haircommander) [reply pkg/client/client.go:1020]: though I don't know if we're using this anywhere so it may not actually matter..

Comment (saschagrunert) [reply pkg/client/client.go:1020]: No right now it’s not being used.

**Pull request merged**